### ISSUE TITLE
Differentiate between full and partial support in no-unsupported-browser-features

### ISSUE THREAD

**Issue by edg2s** [stylelint/stylelint#1401]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

'background-size: contain;' is flagged as unsupported by Chrome 4-14 and Android Browser 4-4.3. According to caniuse these browsers support background image properties fine (e.g. background-clip, background-origin, background-size), just not background-size when using the background shorthand: http://caniuse.com/#search=background-image

> Which rule, if any, is this issue related to?

no-unsupported-browser-features

> What CSS is needed to reproduce this issue?

```css
.foo {
  background-size: contain;
  background-clip: content-box;
  background-origin: content-box;
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "no-unsupported-browser-features": [ true, {
      "browsers": [ "Chrome >= 1", "ChromeAndroid >= 1" ]
    } ]
  }
}
```

> Which version of stylelint are you using?

6.3.3

> How are you running stylelint: CLI, PostCSS plugin, Node API?

Node API (grunt-stylelint)

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

_Unexpected browser feature CSS3 Background-image options not supported by: Chrome (4,5,6,7,8,9,10,11,12,13,14), Android Browser (4,4.1,4.2-4.3) (no-unsupported-browser-features)_

**Comment by davidtheclark** [comment]
You are using `background-size: contain`, which is not as widely supported as `background-image`: http://caniuse.com/#feat=object-fit

**Comment by davidtheclark** [comment]
Please modify the issue to clarify you usage of `background-size` vs. `background-image` —— I think you are using them interchangeably?

**Comment by edg2s** [comment]
Done.

**Comment by edg2s** [comment]
@ntwb Why is this blocked?

**Comment by davidtheclark** [comment]
@edg2s: This will need to be filed upstream with doiuse, which is the tool that this rule uses.

**Comment by jeddy3** [comment]
@edg2s As @ohskylab points out, this is a current limitation of `doiuse`. If you need this feature, then please consider contributing it to the `doiuse` repo.
